CHANGELOG.md

```diff
@@ -1,296 +1,477 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [2.4.0] - 2023-07-15
+
+### Added
+
+- Pagination to the output of the `favorites list` command
+
+### Fixed
+
+- Favorites list exceeding Discord's size limit could not be
+  viewed ([#606](https://github.com/codetheweb/muse/issues/606))
+
 ## [2.3.0] - 2023-05-13
+
 ### Added
+
```

Comment:
I should get this file auto formatted, but let's revert the formatting changes for now